eq-author-api/src/validation/customKeywords/validatePipingInTitle.js

```diff
@@ -0,0 +1,96 @@
+const {
+  PIPING_TITLE_DELETED,
+  PIPING_TITLE_MOVED,
+} = require("../../../constants/validationErrorCodes");
+const cheerio = require("cheerio");
+const getPreviousAnswersForPage = require("../../../src/businessLogic/getPreviousAnswersForPage");
+const { flatMap, compact } = require("lodash/fp");
+
+module.exports = function(ajv) {
+  ajv.addKeyword("validatePipingInTitle", {
+    $data: true,
+    validate: function isValid(
+      otherFields,
+      entityData,
+      fieldValue,
+      dataPath,
+      parentData,
+      fieldName,
+      questionnaire
+    ) {
+      isValid.errors = [];
+
+      const splitDataPath = dataPath.split("/");
+      const currentPage =
+        questionnaire.sections[splitDataPath[2]].pages[splitDataPath[4]];
```

Comment:
More of a note to self, really, but we could move these two lines into a util function and update the other custom keyword functions as I'm pretty sure they all (or most of them) use this.

Reply:
There are 2 similar functions - but one gets id so prob not worth it ?